### PR TITLE
Add base implementation of Telemetry

### PR DESCRIPTION
**Pull request by seriema**

### CODE REVIEW

File: build/database/ProvisionDatabase_MVC.cs [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            try
+            {
+                // Owin is not configured that becuase we have catch (membership provider  there is not problem)
+                ServiceLocator.Current.TryGetExistingInstance<UIUserProvider>(out userProvider);
+                return userProvider;
+            }
+            catch { }
+
+            // in the case of aspnet identity the rpovider is not in the service locator before owin is sets up then we create own.
+            var userManager = new ApplicationUserManager<ApplicationUser>(new UserStore<ApplicationUser>(new ApplicationDbContext<ApplicationUser>("EPiServerDB")));
```
Comment (valdisiljuconoks) [inline build/database/ProvisionDatabase_MVC.cs]: I would extract name of the connectionString from config element. `EPiServerDataStoreSection.Instance.DataSettings.ConnectionStringName`

File: build/packaging/transforms/Alloy.Sample.BlockEnhancements.web.config.transform [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<modifications>
+    <add path="configuration/episerver.framework/virtualRoles/providers">
+        <add __keyAttributes="name" name="EPiBetaUsers" type="EPiServer.Security.AuthenticatedRole, EPiServer.Framework" />
+    </add>
+    <add path="configuration/episerver.framework">
+        <licensing licenseFilePath="E:\License.config" />
```
Comment (valdisiljuconoks) [inline build/packaging/transforms/Alloy.Sample.BlockEnhancements.web.config.transform]: will this work (absolute path to license)?

**Comment by seriema** [comment]
Github wasn't picking up on the rebased branch so we had to re-create the PR here: https://github.com/episerver/EPiServer.Labs.BlockEnhancements/pull/44

File: build/packaging/transforms/Alloy.Sample.BlockEnhancements.web.config.transform (hunk 2 above)
Comment (seriema) [reply build/packaging/transforms/Alloy.Sample.BlockEnhancements.web.config.transform]: This is just the "daily site" system we use to host a new site for every git branch for ease of reviewing internally.

**Pull request closed without merging**